lib/puppet/type.rb

```diff
@@ -1537,13 +1537,42 @@ def to_edges
             :event => self.class.events,
             :callback => method
           }
-          self.debug { "subscribes to #{related_resource.ref}" }
+          ## Corrected syntax of debug statement to reflect the way this was called.
+          # i.e. before, after, subscribe, notify
+          #self.debug { "subscribes to #{related_resource.ref}" }
+	  self.debug do 
```

Comment:
Looks like there's an off-by-one indentation problem here. Also, we don't need to keep around the old debug message as a comment.

Reply:
It looks like I had copied tabs in one of the case statements.
I did replace those and removed the commented old code.

Please do squash the commits with the merge.